ServerCore/Pages/Responses/Delete.cshtml

```diff
@@ -31,10 +31,10 @@
             @Html.DisplayFor(model => model.PuzzleResponse.Note)
         </dd>
     </dl>
-    
+
     <form method="post">
         <input type="hidden" asp-for="PuzzleResponse.ID" />
         <input type="submit" value="Delete" class="btn btn-default" /> |
-        <a asp-page="./Index">Back to List</a>
+        <a asp-page="./Index">Back to Puzzle List</a>
```

Comment:
But this appears to go back to the list of responses? Doesn't it navigate to /{eventId}/Responses/Index?

Reply:
Please change the text of the link to "Back to Response List"

Reply:
Or even better, port this from the Details page: `<a asp-page="./Index" asp-route-puzzleid="@Model.PuzzleId">Back to List</a>`